src/main/scala/templatecontrol/live/LiveGitProject.scala

```diff
@@ -84,7 +84,9 @@ class LiveGitProject(workingDir: File, upstream: GHRepository, remote: GHReposit
                   |$message
                   |```
           """.stripMargin
-    upstream.createPullRequest(title, head, base, body)
+    upstream
+      .createPullRequest(title, head, base, body)
+      .setLabels("merge-when-green", "template-control")
```

Comment:
Are those tags available in the sample repositories? Do we need to create them upfront?

Reply:
I have used a shell script to create the labels with the specific colours. This is done for all Play and Lagom sample projects.